data/wanderer/wanderers middle.txt

```diff
@@ -20,6 +20,394 @@ mission "Wanderers: Pre-Colony"
+	source "Vara K'chrai"
+	destination "Spera Anatrusk"
+	stopover "Varu Tek'kai"
+	cargo "heavy machinery" 185
+
+	to offer
+		has "Wanderers: Spera Anatrusk Outfitter 1: offered"
+
+	on stopover
+		dialog "You notice that each crate of machinery has been signed by the dock team that loaded it."
+
+	on visit
+		dialog "You've arrived at <planet> but the escort carrying the <cargo> hasn't landed. You realize you need to depart and wait for it."
+
+	on abort
+		event "wanderers: spera anatrusk colony" 40
+
+	on fail
+		event "wanderers: spera anatrusk colony" 40
+
+	on complete
+		dialog "The construction supervisor drops by as the workers are unloading. He gives you a credit chip for <payment> and asks you to meet him in the tent that serves as the spaceport when all the deliveries are done."
+		payment 250000
+
+
+mission "Wanderers: Spera Anatrusk Outfitter 3"
+	landing
+	name "Warehouse equipment to <planet>"
+	description "Deliver warehouse equipment and staff for the outfitter on <destination>."
+	source "Spera Anatrusk"
+	destination "Vara K'chrai"
+
+	to offer
+		has "Wanderers: Spera Anatrusk Outfitter 2a: done"
+		has "Wanderers: Spera Anatrusk Outfitter 2b: done"
+		has "Wanderers: Spera Anatrusk Outfitter 2c: done"
+
+	on offer
+		conversation 
+			`This time you land on a concrete pad and you see the familiar hexagonal layout of a Wanderer spaceport. There are tents instead of buildings but you can see stakes outlining the shapes to come. You pass a group of Wanderer captains who appear to be holding a swap meet looking for spare ammunition and equipment so they can avoid the trip back across the Eye.`
+			`	The construction supervisor bobs his head enthusiastically when he sees you and eventually remembers to switch to Hai. "Thank you, Captain <last>," he says. "We would be waiting weeks to get started building without your help. We now need to get the warehouse equipment and [personnel, staff] in to organize the outfitter and [technicians, fixers] to do the [maintenance, repairs]. Would you be able to do this for us as well?`
+			choice
+				`	"Of course. I need an outfitter here as much as anybody."`
+					goto willdo
+				`	"You'll need to find someone else for the next part."`
+					decline
+			label willdo 
+			`	"That is [optimal, excellent]," he says as he hands you a data chip. "If you will take this to <destination>, they will make all the arrangements."`
+				accept
+	
+	on decline
+		event "wanderers: spera anatrusk colony" 30
+
+	on abort
+		event "wanderers: spera anatrusk colony" 30
+
+	on fail
+		event "wanderers: spera anatrusk colony" 30
+
+	on complete
+		conversation
+			`Things go much smoother this time. The Wanderer in the office takes the chip and after a few moments loads their own translation application and says, "[Complete, Done]. On job [board, list] soon. Spaceport if no see."`
```

Comment:
```suggestion
			`Things go much smoother this time. The Wanderer in the office takes the chip and after a few moments loads their own translation application and says, "[Complete, done]. In mission [list, register] soon. Spaceport if no see."`
```
Precedent for unclear translations at the beginning of a sentence looks to be leaving the second word uncapitalized, viewing something like Alpha Surveillance A.

Then also ditto for the above job board edit.